### ISSUE TITLE
Inconsistent use of translations

### ISSUE THREAD

**Issue by ashsmith** [BlueAcornInc/UniversalAnalytics#7]
In `BlueAcorn_UniversalAnalytics_Model_Monitor`, the methods `generateTransactionData`, `filterObjectArray`, `parseObject` are inconsistent.

`generateTransactionData` expects the key value pair for translations to be like so: `magentoAttr => googleAttr`. Whereas the other two methods expect `googleAttr => magentoAttr`.

**Comment by Skrath** [comment]
Yeah, this is leftover from converting a lot of stuff and definitely needs to be cleaned up.

**Comment by mpchadwick** [comment]
The inconsistency is definitely confusing, but changing is likely to break custom work anyone did for these https://github.com/BlueAcornInc/UniversalAnalytics/issues/10 https://github.com/BlueAcornInc/UniversalAnalytics/issues/12
